scrapy/pipelines/images.py

```diff
@@ -100,14 +101,20 @@ def file_downloaded(self, response, request, info):
     def image_downloaded(self, response, request, info):
         checksum = None
         for path, image, buf in self.get_images(response, request, info):
+            content_type = Image.MIME[image.format]
+            if self.convert_type != 'PRESERVE':
```

Comment:
(looks like I missed to comment on that earlier)

I'm not really for using a special `'PRESERVE'` string value to mean "do not convert". I would prefer using `None`.
So `if self.convert_type != 'PRESERVE':` would become `if self.convert_type is not None:` or even `if not self.convert_type:` could work I believe.
Does it make sense? Am I missing something?